test/e2e/scheduler_predicates.go

```diff
@@ -174,8 +177,8 @@ var _ = Describe("SchedulerPredicates", func() {
 
 		allPods, err := c.Pods(api.NamespaceAll).List(labels.Everything(), fields.Everything())
 		expectNoError(err)
-		currentlyRunningPods, currentlyDeadPods := getPodsNumbers(allPods)
-		podsNeededForSaturation := int(totalPodCapacity) - currentlyRunningPods
+		currentlyScheduledPods, currentlyNotScheduledPods := getPodsScheduledNumbers(allPods)
```

Comment:
You should change the description in line 168 above to read "... number of pods that are allowed to schedule?"